qutebrowser/mainwindow/prompt.py

```diff
@@ -467,6 +467,27 @@ def prompt_yank(self, sel=False):
         utils.set_clipboard(question.url, sel)
         message.info("Yanked to {}: {}".format(target, question.url))
 
+    @cmdutils.register(
+        instance='prompt-container', scope='window',
+        modes=[usertypes.KeyMode.prompt])
+    def prompt_external_picker(self):
```

Comment:
We call this `fileselect` so far, e.g. in the settings. Should the command be named something like `prompt_fileselect_external` or so too?

Reply:
Done in 65c3007fc07f708e716c9cb14fefd10bf5f240d7